curriculum/challenges/english/20-upcoming-python/learn-lambda-functions-by-building-a-expense-tracker/657eb8e0d19d7d50509c1723.md

```diff
@@ -0,0 +1,64 @@
+---
+id: 657eb8e0d19d7d50509c1723
+title: Step 16
+challengeType: 20
+dashedName: step-16
+---
+
+# --description--
+
+To add an expense, we use an if statement to check the user's choice. If the choice is '1', it means the user wants to add an expense.
+
+Still in the While loop, under the `choice` input, write an if statement to check if choice equals '1'. If it's true, it will be the starting point for adding a new expense. The if statement should contain a pass statement for now.
```

Comment:
```suggestion
Still in the while loop, under the `choice` input, write an `if` statement to check if choice equals '1'. If it's true, it will be the starting point for adding a new expense. The `if` statement should contain a `pass` statement for now.
```